_content/articles/sermon-on-the-letting-in-and-not-letting-in_ernst_waldschmidt.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Mahāmaudgalyāyana's Sermon on the Letting-in and Not Letting-in (of Sensitive Influences)"
+authors:
+  - "Ernst Waldschmidt"
+external_url: "https://journals.ub.uni-heidelberg.de/index.php/jiabs/article/view/8453/2360"
+drive_links:
+  - "https://drive.google.com/file/d/1sVIf327n20H0n_asX3wD4jPPYhJisvXS/view?usp=sharing"
+course: problems
+tags:
+  - sense-restraint
```

Comment:
let's add the `sa` tag for the Sumyakta Agama (this sutra is sa1176) and `manuscripts` as well